packages/core/hooks/useOpenWithMenuItems/index.tsx

```diff
@@ -54,9 +54,9 @@ const WEB_APPS = (fileDetails?: FileDetail): WebApps => ({
 const DESKTOP_APPS = (fileDetails?: FileDetail): { agave: IContextualMenuItem } => ({
     agave: {
         key: "agave",
-        className: styles.agaveLink,
+        className: styles.desktopMenuItem,
         text: "AGAVE",
-        title: `Open files with AGAVE`,
+        title: "Open files with AGAVE v1.7.2+",
```

Comment:
Do we have any concern with keeping this version number synced?

Reply:
Nah because unless they remove support for opening from a browser (pretty unlikely) the user just needs a version >= when they added it when was 1.7.2